src/amm.rs

```diff
@@ -116,7 +117,8 @@ impl Web3 {
                     "Bad response from swap price".to_string(),
                 ))
             }
-        }))
+        })
+        .unwrap())
```

Comment:
`from_bytes_be` can fail if there are more than 32 bytes (and `num256` would just panic internally instead of giving a heads up). There is a checked `get(0..32)` range in the previous lines so it is ok